velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -191,6 +192,17 @@ TEST_F(DateTimeFunctionsTest, year) {
   EXPECT_EQ(2001, year(Timestamp(998423705, 321000000)));
 }
 
+TEST_F(DateTimeFunctionsTest, yearDate) {
+  const auto year = [&](std::optional<Date> date) {
```

Comment:
nit: make const and pass by reference (here and below)

Reply:
same as above